src/lib/utils.js

```diff
@@ -1,19 +1,21 @@
 'use strict'
 
-module.exports = class Utils {
-  /**
-   * Get all possible pair combinations from an array.
-   *
-   * Example:
-   *   getPairs ([1, 2, 3, 4])
-   *   // => [ [ 1, 2 ], [ 1, 3 ], [ 1, 4 ], [ 2, 3 ], [ 2, 4 ], [ 3, 4 ] ]
-   *
-   * @param {array} arr Input array
-   * @return {array[]} Possible pairs
-   */
-  static getPairs (arr) {
-    return arr.reduce((prev, cur, i) => (
-      prev.concat(arr.slice(i + 1).map((val) => [cur, val]))
-    ), [])
-  }
+/**
```

Comment:
This is also changed in #125: https://github.com/interledger/five-bells-connector/pull/125/files#diff-fb54f78b07de422efaa3d2d82bb82db3L3.